inc/config/dynamic-datasource-trait.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace RemoteDataBlocks\Config;
+
+trait DynamicDatasource {
```

Comment:
How do we all feel about traits?

Reply:
This was mainly a shortcut since these values didn't make much sense in an `HttpDatasource` instance.

In a world with only Single Inheritance, traits can be handy, but sometimes confusing.